zconf-ng.h.in

```diff
@@ -59,7 +59,7 @@
  * This is not mandatory, but it offers a little performance increase.
  */
 #if defined(ZLIB_DLL) && (defined(_WIN32) || (__has_declspec_attribute(dllexport) && __has_declspec_attribute(dllimport)))
-#  ifdef ZLIB_INTERNAL
+#  ifdef ZINTERNAL
```

Comment:
Unsure about this one too since it is in `zconf-ng.h`.

Reply:
Same as the one in zlib.h, so I think this is OK.